src/eventmaster/api_handlers.go

```diff
@@ -143,12 +143,12 @@ func (h *httpHandler) sendError(w http.ResponseWriter, code int, err error, mess
 	w.Write([]byte(errMsg))
 }
 
-func (h *httpHandler) sendResp(w http.ResponseWriter, key string, val string, path string) {
+func (h *httpHandler) sendResp(w http.ResponseWriter, key string, val interface{}, path string) {
```

Comment:
There is a switch on key below  which is what I was getting at. We might want to have "val" be the map-- so it isn't switched here.

This way we could also send back a string or w/e without having to modify this method.